UPGRADE.md

```diff
@@ -199,6 +199,14 @@ These services will be generated automatically based on subject name.
 
 ### ShopBundle
 
+* The following templates were moved:
+
+  * `@SyliusShop/Homepage/_header.html.twig` -> `@SyliusShop/_header.html.twig`
+  * `@SyliusShop/Homepage/_footer.html.twig` -> `@SyliusShop/_footer.html.twig`
+  * `@SyliusShop/Homepage/Menu/_security.html.twig` -> `@SyliusShop/Menu/_security.html.twig`
+  * `@SyliusShop/_currencySwitch.html.twig` -> `@SyliusShop/Menu/_currencySwitch.html.twig`
+  * `@SyliusShop/_localeSwitch.html.twig` -> `@SyliusShop/Menu/_localeSwitch.html.twig`
```

Comment:
These two widgets have the same responsibility as the security widget, so putting them into `Menu` directory together makes sense for me.